python/src/trezorlib/cli/btc.py

```diff
@@ -49,12 +56,27 @@
     "tr": messages.OutputScriptType.PAYTOTAPROOT,
 }
 
-BIP_PURPOSE_TO_SCRIPT_TYPE = {
-    tools.H_(44): messages.InputScriptType.SPENDADDRESS,
-    tools.H_(49): messages.InputScriptType.SPENDP2SHWITNESS,
-    tools.H_(84): messages.InputScriptType.SPENDWITNESS,
-    tools.H_(86): messages.InputScriptType.SPENDTAPROOT,
-    tools.H_(10025): messages.InputScriptType.SPENDTAPROOT,
+BIP_PURPOSE_TO_DEFAULT_SCRIPT_TYPE = {
+    PURPOSE_BIP44: messages.InputScriptType.SPENDADDRESS,
+    PURPOSE_BIP49: messages.InputScriptType.SPENDP2SHWITNESS,
+    PURPOSE_BIP84: messages.InputScriptType.SPENDWITNESS,
+    PURPOSE_BIP86: messages.InputScriptType.SPENDTAPROOT,
+    PURPOSE_SLIP25: messages.InputScriptType.SPENDTAPROOT,
+}
```

Comment:
Note that `BIP_PURPOSE_TO_DEFAULT_SCRIPT_TYPE` could be constructed from `SCRIPT_TYPE_TO_BIP_PURPOSES` right now, but if the latter becomes more complex, e.g. `messages.InputScriptType.SPENDWITNESS: (PURPOSE_BIP84, PURPOSE_SLIP25),` then it's won't be clear what the default for `PURPOSE_SLIP25` should be.